definitions/v4/drunkenslug.yml

```diff
@@ -0,0 +1,164 @@
+---
+id: drunkenslug-yml
+name: DrunkenSlug (YML)
+description: "DrunkenSlug is a Private Usenet Indexer"
+language: en-US
+type: private
+redirect: false
```

Comment:
All of these should support redirecting, basically anything that can provide a download link that doesn't require auth. In this case all you need is an APIKEY in the query, thus any NEWZNAB indexer is eligible to allow redirect if user wants it.